.github/workflows/test-recent-optimism-block.yml

```diff
@@ -0,0 +1,50 @@
+name: Run a recent full Optimism block in the Hardhat Network
+
+on:
+  schedule:
+    - cron: "0 */8 * * *"
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: packages/hardhat-core
+
+jobs:
+  test-recent-optimism-block:
+    name: Test recent Optimism block
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust (stable)
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          override: true
+
+      - uses: Swatinem/rust-cache@v2
+
+      - name: Cache EDR RPC cache
+        uses: actions/cache@v2
```

Comment:
```suggestion
        uses: actions/cache@v4
```

Reply:
Thanks for detecting these!

Do you have an automated way of linting GitHub actions for outdated versions while editing? My VSCode extension didn't highlight these issues